vars/dockerExecute.groovy

```diff
@@ -13,28 +14,91 @@ import groovy.transform.Field
 @Field def PLUGIN_ID_DOCKER_WORKFLOW = 'docker-workflow'
 
 @Field Set GENERAL_CONFIG_KEYS = [
+    /**
+     *
```

Comment:
I guess here should be some text

Reply:
It was missing before, we can add this with a separate PR.